### ISSUE TITLE
AwarenessNotice for diffrent senses

### ISSUE THREAD

**Issue by drowzero0** [OctarineSourcerer/cyberpunk2020-foundry#52]
Since a few types of Cyberware can modify your AwarenessNotice check for different senses it would be nice to check for different senses:
Like creating awareness notice as a group but with a value.
+ a list of the senses that awareness/notice can be modified with.
hearing
seeing
feeling
smelling

Like checking on visual awareness notice and having an ocular cyberupgrade.
Or Cyberears for a check on hearing. and so on.

this would make the implementation of cyberware much easier since you do not need an extra field in cyberware.sheet for modifiers which can affect different skills/stats

**Comment by OctarineSourcerer** [comment]
This gets a little gnarly, as by default awareness/notice is a fairly all-encompassing stat, and having it split like that by default would confuse people, especially new players. I'd propose a more generic method: I intend to have modifier dialogs appear for when rolling skills or stats, that the player can choose to show (or skip) by holding a key when they roll. This modifier box should perhaps list cyberware targeting that skill or stat, and enable/disable them, or just in AwarenessNotice's case allow for choosing a sense to do it with (similar with how when shooting you choose a firemode).

How does that sound?

**Comment by drowzero0** [comment]
sounds like a good idea.
was just mentioning it, since i am inserting some cyberware atm and saw those modifiers.
a choice while clicking a roll would be far more appealing.

**Comment by OctarineSourcerer** [comment]
Yep, having a modifier window like that is super useful, and is in a lot of systems for a reason :) I'll change the title of the issue to reflect what we're going for here.
